openpathsampling/analysis/shooting_point_analysis.py

```diff
@@ -91,11 +91,16 @@ class ShootingPointAnalysis(SimpleProgress, SnapshotByCoordinateDict):
     states : list of :class:`.Volume`
         volumes to consider as states for the analysis. For pandas output,
         these volumes must be named.
+    skip_rejected: bool, default False
+         boolean flag to skip rejected moves
     """
-    def __init__(self, steps, states):
+    def __init__(self, steps, states, skip_rejected=False):
         super(ShootingPointAnalysis, self).__init__()
         self.states = states
-        if steps is not None:
+        self.skip_rejected = skip_rejected
+        if steps and self.skip_rejected:
+            steps = [step for step in steps if step.change.canonical.accepted]
+        if steps:
```

Comment:
changed this from `is not None` to test against `truthy`, which seems like a robuster approach to catch empty lists etc early